internal/services/appservice/function_app_flex_consumption_resource_test.go

```diff
@@ -1230,6 +1249,66 @@ resource "azurerm_function_app_flex_consumption" "test" {
 `, r.template(data), data.RandomInteger)
 }
 
+func (r FunctionAppFlexConsumptionResource) vNetIntegration_subnetWithVnetProperties(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+%s
+
+resource "azurerm_virtual_network" "test" {
+  name                = "vnet-%d"
```

Comment:
could we prefix this with `acctest`?

```suggestion
  name                = "acctest-vnet-%d"
```

Reply:
updated as per comments